src/Adapter/MSTest.TestAdapter/Execution/TestMethodRunner.cs

```diff
@@ -170,7 +170,7 @@ internal TestResult[] RunTestMethod()
         var parentStopwatch = Stopwatch.StartNew();
         if (_test.DataType == DynamicDataType.ITestDataSource)
         {
-            if (_test.TestDataSourceIgnoreMessage is not null)
+            if (!string.IsNullOrEmpty(_test.TestDataSourceIgnoreMessage))
```

Comment:
Shall we also exclude full whitespace?

Reply:
@Evangelink I thought about it, but I think if the user explicitly sets ignore message to `"  "`, then he wants to ignore. It's unlikely someone will do it though.

I actually think that the check for empty is a workaround for VSTest bug and I think it should be allowed to set to empty string and have the test ignored.